### PR TITLE
feat: Print 123

### LINKED ISSUE #2: Do print 123
화면에 123 출력

## 할 일

- [x] 화면에 123을 출력시키겠습니다.

### PR DESCRIPTION
**Pull request by icblue21**
Done print123 

resolve #2 
resolves resolved

fix
close

### CODE REVIEW

File: print_hello.py [hunk 1]
```diff
@@ -1 +1 @@
-## Print Hello
+print(123)
```
Comment (724thomas) [inline print_hello.py]: 123이 아닌 hello로 수정바람

**Review by 724thomas — CHANGES_REQUESTED** [review]
123이 아닌 hello로 수정바람

File: print_hello.py (hunk 1 above)
Comment (icblue21) [reply print_hello.py]: OK

**Review by 724thomas — APPROVED** [review]
굿~

**Pull request merged**